ui/react-app/web.go

```diff
@@ -0,0 +1,51 @@
+package reactApp
```

Comment:
All code in this file is purely extracted from the old `ui/web.go`. No substantive changes.